newclient/src/modules/address/components/TableAddressTokensHeader.vue

```diff
@@ -37,23 +42,27 @@
                     <v-flex v-if="hasTokens" d-flex align-center>
                         <v-layout align-start justify-right>
                             <v-btn
-                                v-if="!loading && !isTokenSortActive(1)"
+                                v-if="!loading && !isTokenSortActive(0)"
                                 :class="[!isTokenSortActive(0) && !isTokenSortActive(1) ? 'inactive-btn' : '']"
                                 flat
                                 icon
-                                @click="selectTokenFilter(0)"
+                                @click="selectTokenFilter(1)"
```

Comment:
not sure if we want to change the other token sorts to match selectTokenFilter(oddNum) or vice versa

Reply:
Hey this was done intentionally to change the icon :)